docs/website/docs/walkthroughs/zendesk-weaviate.md

```diff
@@ -0,0 +1,179 @@
+
+In this walkthrough, we’ll show you how to import Zendesk ticket data to one of the vector databases, Weaviate. We’ll use dlt to connect to the Zendesk API, extract the data, and load it into Weaviate.
+
+## Prerequisites
+
+We're going to use some ready-made components from the [dlt ecosystem](https://dlthub.com/docs/dlt-ecosystem) to make this process easier:
+
+1. A [Zendesk verified source](../dlt-ecosystem/verified-sources/zendesk.md) to extract the tickets from the API.
+2. A [Weaviate destination](../dlt-ecosystem/destinations/weaviate.md) to load the data into a Weaviate instance.
+
+## Setup
+
+1. Create a new folder for your project, navigate to it, and create a virtual environment:
+
+    ```bash
+    mkdir zendesk-weaviate
+    cd zendesk-weaviate
+    python -m venv venv
+    source venv/bin/activate
+    ```
+2. Install dlt with Weaviate support
+
+    ```bash
+    pip install "dlt[weaviate]"
+    ```
+
+3. Install dlt Zendesk verified source
+
+    ```bash
+    dlt init zendesk weaviate
+    ```
+
+The last command [dlt init](../reference/command-line-interface#dlt-init) initializes dlt project: it downloads the verified source and installs it in your project folder.
+
+## Configuration
+
+### Source
+
+Before we configure the source and destination, you need to make sure you have access to API in both Zendesk and Weaviate.
+
+Head to the [Zendesk](../dlt-ecosystem/verified-sources/zendesk.md) docs to see how to fetch credentials for Zendesk API. In our walkthrough, we're using the email address and password authentication method. Once you have fetched the credentials, you can configure the source. Add the following lines to the dlt secrets file `~/.dlt/secrets.toml`:
+
+```toml
+[sources.zendesk.zendesk_support.credentials]
+password = "..."
+subdomain = "..."
+email = "..."
+```
+
+### Destination
+
+For the destination we're using [Weaviate Cloud Services](https://console.weaviate.cloud/). You would need to create an account and get a URL and an API key for your Weaviate instance. We're also be using [OpenAI API](https://platform.openai.com/) to generate embeddings for the text data needed to perform vector search. If you haven't already, you would need to create an account and get an API key for OpenAI API.
+
+When you have the credentials, add more lines to the dlt secrets file `~/.dlt/secrets.toml`:
+
+```toml
+[destination.weaviate.credentials]
+url = "https://weaviate_url"
+api_key = "api_key"
+
+[destination.weaviate.credentials.additional_headers]
+X-OpenAI-Api-Key = "sk-..
+```
+
+### Configuring the pipeline
+
+When you run `dlt init zendesk weaviate`, dlt creates a file called `zendesk_pipeline.py` in the current directory. This file contains an example pipeline that you can use to load data from Zendesk source. Let's edit this file to make it work for our use case:
+
```

Comment:
you should summarize the most important changes you've made during customization. for sure explain the use of the adapter and that you are selecting only `tickets` resource to load.

and why "subject" and "description" are vectorized?